content/graphqlschema/6.md

```diff
@@ -2,40 +2,33 @@
 date = "2017-05-01T10:41:33+10:00"
 next = "/graphqlschema/7"
 prev = "/graphqlschema/5"
-title = "Exercise : Integrating existing data"
+title = "Exercise: Integrating existing data"
```

Comment:
If we have liberty to deviate from a direct mapping of the DQL tour content, then this lesson could be removed. We have already shown the content of this course multiple times already. This is easier understood in GraphQL than it is in DQL